apps/storefront/src/pages/Login/index.tsx

```diff
@@ -443,7 +436,8 @@ export default function Login(props: PageProps) {
                           }}
                         >
                           <LoginPanel
-                            loginInfo={loginInfo}
+                            createAccountButtonText={loginInfo.createAccountButtonText}
+                            widgetBodyText={loginInfo.widgetBodyText}
```

Comment:
This ensures only necessary data is shared between components. It also helps with re-renders.